node/pkg/reporter/app.go

```diff
@@ -14,7 +14,8 @@ import (
 
 func New() *App {
 	return &App{
-		Reporters: []*Reporter{},
+		Reporters:  []*Reporter{},
+		LatestData: &sync.Map{},
```

Comment:
Personally prefer this way since it is more explicit 😅
```suggestion
		LatestData: new(sync.Map),
```